procurement_jit_assign_move/models/stock_move.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.multi
+    def action_confirm(self):
+        res = super(StockMove, self).action_confirm()
+        make_to_stock_moves = self.filtered(
+            lambda x:
+            x.procurement_id.rule_id.procure_method == 'make_to_stock'
```

Comment:
Why not use x.procure_method immediately?  It is different from the procurement_jit in 9 as that one only does it for sale orders, not for just any stock move.  I guess it depends on what you want to achieve.